### ISSUE TITLE
battery_time overlaps label after setting 3+ columns

### ISSUE THREAD

**Issue by mc711** [flightlessmango/MangoHud#1162]
**Describe the bug**
battery_time overlaps BAT label if table_columns>3

**List relevant hardware/software information**
- Linux Distribution SteamOS 3.5.1
- MangoHud version 0.7.0
- GPU Steam Deck APU

**To Reproduce**
create custom mangohud.cfg
i have attached my cfg to reproduce (remove .txt github does not allow .cfg upload)
[MangoHud.conf.txt](https://github.com/flightlessmango/MangoHud/files/13223572/MangoHud.conf.txt)

**Expected behavior**
battery_time should extend to extra column not overlap label

**Screenshots**
![1](https://github.com/flightlessmango/MangoHud/assets/2786469/3c232431-1e35-4660-ab57-1287b0027617)
![2](https://github.com/flightlessmango/MangoHud/assets/2786469/0a310d62-e25c-401d-a14f-c354ca0ff65b)

**Additional context**

**Comment by mfrh89** [comment]
I have the same/an similar issue in **horizontal** mode with **hud_compact** setting:

![image](https://github.com/flightlessmango/MangoHud/assets/88285794/be9f12dc-86b3-4ca8-938b-6e8ba8e9f087)

![image](https://github.com/flightlessmango/MangoHud/assets/88285794/7558df31-7aee-4a5b-b450-2ac01e63e5f8)


Only happens when **battery_time** is added + affects also the **time** item.

**Comment by bass3l** [comment]
Facing the same problem where I'm trying to only show the battery information, also is there a way to hide the label? like `battery_no_label` ? 

![Screenshot_20240430_201554](https://github.com/flightlessmango/MangoHud/assets/9248745/f0b6959d-f04d-4a55-b753-9299623b5da7)

**Comment by syrnyk** [comment]
Yeah, same issue on Steam Deck with horizontal mode + battery time and actual time. Both stats are wrapped at the start and overlap, no matter where I place them on the HUD

**Comment by tastypurgen** [comment]
same problem in horizontal mode with compact hud
